apps/vs-code-designer/src/app/commands/generateDeploymentScripts/__test__/generateDeploymentScripts.test.ts

```diff
@@ -0,0 +1,190 @@
+import { describe, expect, beforeEach, vi, it, Mock } from 'vitest';
+import * as vscode from 'vscode';
+import { generateDeploymentScripts } from '../generateDeploymentScripts';
+import { IActionContext, UserCancelledError } from '@microsoft/vscode-azext-utils';
+import { ext } from '../../../../extensionVariables';
+import { localize } from '../../../../localize';
+import { addLocalFuncTelemetry } from '../../../utils/funcCoreTools/funcVersion';
+import { convertToWorkspace } from '../../createNewCodeProject/CodeProjectBase/ConvertToWorkspace';
+import { getWorkspaceFolder, isMultiRootWorkspace } from '../../../utils/workspace';
+import { isLogicAppProject, tryGetLogicAppProjectRoot } from '../../../utils/verifyIsProject';
+import { AzureWizard } from '@microsoft/vscode-azext-utils';
+import { COMMON_ERRORS } from '../../../../constants';
+
+vi.mock('@microsoft/vscode-azext-utils', () => {
```

Comment:
NIT: I wonder if these mocks from `@microsoft/vscode-azext-utils` can be passed to the apps/vs-code-designer/test-setup.ts. So we can reuse them and not rewrite them